src/dealDashboard/dealVotes/dealVotes.ts

```diff
@@ -1,36 +1,73 @@
-import { autoinject, bindable } from "aurelia-framework";
+import { autoinject, bindable, computedFrom } from "aurelia-framework";
 import { DealTokenSwap } from "entities/DealTokenSwap";
 import "./dealVotes.scss";
 import { Router } from "aurelia-router";
 import { EthereumService } from "../../services/EthereumService";
-import { AlertService } from "../../services/AlertService";
+import { FundingModal } from "./fundingModal/fundingModal";
+import { DialogService } from "../../services/DialogService";
 
 @autoinject
 export class DealVotes {
   @bindable deal: DealTokenSwap;
 
   fundingModal: HTMLElement;
 
+  everyTextCopy = [
+    {
+      condition: () => !this.deal.majorityHasVoted && !this.deal.representativeVoted(this.ethereumService.defaultAccountAddress),
+      voteText: "You’re eligible to vote. The deal will be approved once the majority accepts the deal",
+      statusText: "Voting in progress",
+    },
+    {
+      condition: () => !this.deal.majorityHasVoted && this.deal.representativeVoted(this.ethereumService.defaultAccountAddress) && this.deal.isTheProposalLead(this.ethereumService.defaultAccountAddress),
```

Comment:
isVotingInProgressProposalLead

(personally, I would break camelCase pattern and do `isVotingInProgress_ProposalLead`